content/zh/docs/what-is-opentelemetry.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 什么是 OpenTelemetry？
+description: 简短说明 OpenTelemetry 是什么，不是什么。
+aliases: [/zh/about, /zh/docs/concepts/what-is-opentelemetry, /zh/otel]
```

Comment:
For the same reasons given earlier, I'd drop all of these aliases, though I wonder if there'd be value in keeping `/zh/about`. WDYT @svrnm?

```suggestion
```